test/cli-test.js

```diff
@@ -0,0 +1,70 @@
+/* global describe, it */
+var assert = require('assert').strict
+const { exec } = require('child_process')
+
+function cli (command) {
+  return new Promise((resolve, reject) => {
+    exec(command, (error, stdout, stderr) => {
+      if (error) {
+        reject(error)
+      } else {
+        resolve({
+          exitCode: 0,
+          out: JSON.parse(stdout)
+        })
+      }
+    })
+  })
+}
+
+var tests = [
+  /* {
+  c: `node ${__dirname}/../lib/cli.js "kind" ${__dirname}/fixture/activities.json`,
+  e: {
+    exitCode: 0,
+    out: {
+      kind: 'plus#activity'
+    }
+  }
+  }, */
+  {
+    c: `cat ${__dirname}/fixture/activities.json | node ${__dirname}/../bin/json-mask.js "kind"`,
```

Comment:
I didn't catch this before, but what will we do with our Windows friends? I'm concerned that they won't be able to run this test. Are there any alternatives?

Reply:
I'll rewrite the tests with `echo` piping, should work on Windows too.
I'll also add Windows build env to travis.